HW_2/task_3.py

```diff
@@ -0,0 +1,41 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем
+# и знаменателем. Программа должна возвращать сумму и произведение* дробей.
+# Для проверки своего кода используйте модуль fractions.
+import fractions
+
+
+def parse_fraction(fraction_str):
+    parts = fraction_str.split('/')
+    numerator = int(parts[0])
+    denominator = int(parts[1])
+    return numerator, denominator
```

Comment:
Можно значительно проще
```suggestion
    return map(int, fraction_str.split('/'))
```
Даже функцию отдельно писать не нужно